util/src/main/java/io/camunda/zeebe/util/sched/Actor.java

```diff
@@ -10,19 +10,38 @@
 import io.camunda.zeebe.util.CloseableSilently;
 import io.camunda.zeebe.util.Loggers;
 import io.camunda.zeebe.util.sched.future.ActorFuture;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
 public abstract class Actor implements CloseableSilently, AsyncClosable, ConcurrencyControl {
 
+  public static final String ACTOR_PROP_NAME = "actor-name";
+  public static final String ACTOR_PROP_PARTITION_ID = "partitionId";
```

Comment:
:question: I don't think this is used?

Reply:
not yet I feel you haven't read the PR description :sweat_smile: 

> @npepinpe please take a look at this and let me know whether this approach makes sense to you. If this is the case I will change sub classes of the Actor class to add there partition id to the context.

Reply:
Lol I actually re-read it to see if the partition ID was mentioned, don't know how I missed that :see_no_evil: